src/org/labkey/test/tests/JUnitTest.java

```diff
@@ -457,4 +467,35 @@ static void log(String str, PrintStream printStream)
         String d = new SimpleDateFormat("HH:mm:ss,SSS").format(new Date());      // Include time with log entry.  Use format that matches labkey log.
         printStream.println(d + " " + str);
     }
+
+    public static class BaseJUnitTestWrapper extends BaseWebDriverTest
```

Comment:
this makes sense right now, but could probably use some commenting for our future selves to remember why this was needed

Reply:
Yeah. There's a backlog item for general doc improvements for the test runner.